ext/date/date_core.c

```diff
@@ -6574,8 +6589,7 @@ d_lite_upto(VALUE self, VALUE max)
 
 /*
  * call-seq:
- *    d.downto(min)              ->  enumerator
- *    d.downto(min){|date| ...}  ->  self
+ *   downto(min){|date| ... } -> self
  *
  * This method is equivalent to step(min, -1){|date| ...}.
```

Comment:
Should this description also be updated to be consistent with `upto`?

Reply:
Fixed.